proto/forch_configuration.proto

```diff
@@ -28,6 +28,9 @@ message ForchConfig {
 
   // configuration for port proxy server
   ProxyServerConfig proxy_server = 7;
+
+  // configuration for switches monitoring
+  SwitchMonitoring switch_monitoring = 8;
```

Comment:
Seems like this should be a vlan map, and then one of the fields would be packet threshold for that vlan?

Reply:
Yes, done